src/batched/KokkosBatched_InverseLU_Serial_Impl.hpp

```diff
@@ -0,0 +1,195 @@
+#ifndef __KOKKOSBATCHED_INVERSELU_SERIAL_IMPL_HPP__
+#define __KOKKOSBATCHED_INVERSELU_SERIAL_IMPL_HPP__
+
+
+/// \author Vinh Dang (vqdang@sandia.gov)
+
+#include "KokkosBatched_Util.hpp"
+#include "KokkosBatched_Trsm_Decl.hpp"
+#include "KokkosBatched_Trsm_Serial_Impl.hpp"
+
+
+namespace KokkosBatched {
+  namespace Experimental {
+    ///
+    /// Serial Impl
+    /// =========
+
+    ///
+    /// InverseLU no piv
+    ///
+
+#if                                                     \
+  defined(__KOKKOSBATCHED_INTEL_MKL__) &&               \
+  defined(__KOKKOSBATCHED_INTEL_MKL_BATCHED__) &&       \
+  defined(__KOKKOSBATCHED_INTEL_MKL_COMPACT_BATCHED__)
+    template<>
+    template<typename AViewType,
+             typename WViewType>
+    KOKKOS_INLINE_FUNCTION
+    int
+    SerialInverseLU<Algo::InverseLU::CompactMKL>::
+    invoke(const AViewType &A,
+           const WViewType &W,
+           const typename MagnitudeScalarType<typename AViewType::non_const_value_type>::type tiny) {
+      typedef typename AViewType::value_type vector_type;
+      //typedef typename vector_type::value_type value_type;
+
+      const int
+        m = A.dimension(0),
```

Comment:
The `dimension(n)` methods are deprecated; use `extent(n)` instead.

Reply:
@mhoemmen Thanks. They are fixed.